tests/console/cups.pm

```diff
@@ -20,7 +20,6 @@ use version_utils 'is_jeos';
 
 sub run {
     my $self = shift;
-    $self->select_serial_terminal;
```

Comment:
Ehm, as the serial console is significantly faster, this seems like a downgrade for me. Would be better to disable it only for specific reason (eg. arch s390x)? As far as I know, it runs well in other cases.

Reply:
Updated patch to only disable serial on s390